activate.sh

```diff
@@ -24,7 +24,7 @@ cp -v $manifest_dir/private.yaml $kube_dir
 
 # Make sure that the controller node looks for the local pause image
 # TODO: remove this as soon as possible. As an idea, we could use a systemd drop-in unit.
-sed -i 's|--config=/etc/kubernetes/manifests|--config=/etc/kubernetes/manifests --pod-infra-container-image=sles12/pause:1.0.0|g' /etc/kubernetes/kubelet
+sed -i '/^KUBELET_ARGS=/ s/=.*/="--pod-manifest-path=\/etc\/kubernetes\/manifests --pod-infra-container-image=sles12\/pause:1.0.0"/' /etc/kubernetes/kubelet
```

Comment:
The problem that I see with this one is that it's destructive. It will remove other flags set:

```
~ echo "KUBELET_ARGS=-v2" | sed '/^KUBELET_ARGS=/ s/=.*/="--pod-manifest-path=\/etc\/kubernetes\/manifests --pod-infra-container-image=sles12\/pause:1.0.0"/'
KUBELET_ARGS="--pod-manifest-path=/etc/kubernetes/manifests --pod-infra-container-image=sles12/pause:1.0.0"
```

What about something like?

```
~ echo "KUBELET_ARGS=\"-v2\"" | sed 's#^KUBELET_ARGS="#KUBELET_ARGS="--pod-manifest-path=/etc/kubernetes/manifests --pod-infra-container-image=sles12/pause:1.0.0 #'
KUBELET_ARGS="--pod-manifest-path=/etc/kubernetes/manifests --pod-infra-container-image=sles12/pause:1.0.0 -v2"
```